src/crypto/backup.ts

```diff
@@ -474,6 +475,9 @@ export class BackupManager {
                             (<MatrixError>err).data.errcode == "M_NOT_FOUND" ||
                             (<MatrixError>err).data.errcode == "M_WRONG_ROOM_KEYS_VERSION"
                         ) {
+                            // fix do it now as the await check might trigger a new call to the loop before
```

Comment:
"fix do it now"?

Reply:
yeah the existing code as a problem, it's calling checkKeybackup() just there, and this will re-enable backup and start the loop;  but it will do nothing because sendingBackups is true, and anyhow it throws after and the finally reset the sendingBackups to false :/
=> I have a cleaner fix for it stashed but I am a bit reluctant to touch old code too much.
I'll push a commit after and let you review

Reply:
done here => https://github.com/matrix-org/matrix-js-sdk/pull/3652/commits/cdde54af4eea0b4dfc5e4b0ddd8ce5150bb18550